wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/WfsFilterDelegate.java

```diff
@@ -805,18 +849,42 @@ public FilterType propertyIsBetween(String propertyName, long lowerBoundary, lon
 
     @Override
     public FilterType propertyIsBetween(String propertyName, float lowerBoundary,
-            float upperBoundary) {
+                                        float upperBoundary) {
         return buildPropertyIsBetweenFilterType(propertyName, lowerBoundary, upperBoundary);
     }
 
     @Override
     public FilterType propertyIsBetween(String propertyName, double lowerBoundary,
-            double upperBoundary) {
+                                        double upperBoundary) {
         return buildPropertyIsBetweenFilterType(propertyName, lowerBoundary, upperBoundary);
     }
 
+    protected boolean isAfterFilter(FilterType filter) {
+        boolean afterFilter = false;
+        if (isFilterSet(filter)) {
+            return TEMPORAL_OPERATORS.After.equals(filter.getTemporalOps().getName().getLocalPart());
+        }
+        return afterFilter;
```

Comment:
It might be clearer to add an else that returns false instead of making the the boolean variable.